bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/messages.properties

```diff
@@ -1,3 +1,3 @@
-LaunchBarListener_TargetChanged_Msg=IDF launch target has changed for the project. Do you want to delete the `build` folder for the project?
+LaunchBarListener_TargetChanged_Msg=Current target for the project {0} has changed from {1} to {2} in the launchbar, would you like to delete the "build" folder for the project?
```

Comment:
LGTM. Just thinking who would be the behavior in the case of the custom build folder? Are we handling the custom build folder here - for example, deleting the folder?

Reply:
I think that part is already handled in the LaunchBarListener so no issues there.